### DISCUSSION TITLE
Material UI 4 and Storybook

### DISCUSSION THREAD

**Discussion by judicaelandria** [storybookjs/storybook#16282] [Help]
Hey, I have a little bug when using material ui v4 with storybook because he can't find my theme
![error](https://user-images.githubusercontent.com/46365844/136505029-0a9b33b5-cc88-447e-b256-ed8852ee229c.png)

**Comment by LongLiveCHIEF** [comment]
👋  @judicaelandria 

This is solved by wrapping your story with a decorator as described in [Docs - Decorators - context for mocking](https://storybook.js.org/docs/react/writing-stories/decorators#context-for-mocking).  I prefer to inject my theme globally to all my stories by exporting this decorator in the global storybook config by modifying the `.storybook/preview.js` like so:

```jsx
import { ThemeProvider } from '@material-ui/core/styles'
import { myCustomTheme } from '../My/Custom/Theme/Src/Path'

// ...existing config

export const decorators = [
  (Story) => (
    <ThemeProvider theme={myCustomTheme}>
      <Story />
    </ThemeProvider>
  )
]
```

**Reply by judicaelandria** [reply]
Yes I did that but I'm still getting this error
![image](https://user-images.githubusercontent.com/46365844/136908371-c6becb0e-a5c9-46b4-a8ba-b98cbc765268.jpeg)
